UnitsNet/UnitDefinitions/Irradiation.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "Name": "Irradiation",
+  "BaseUnit": "JoulePerSquareMeter",
+  "XmlDoc": "Irradiation is the process by which an object is exposed to radiation. The exposure can originate from various sources, including natural sources.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Irradiation",
+  "Units": [
+    {
+      "SingularName": "JoulePerSquareMeter",
+      "PluralName": "JoulesPerSquareMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "J/m²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattHourPerSquareMeter",
+      "PluralName": "WattHoursPerSquareMeter",
+      "FromUnitToBaseFunc": "x*3600d",
+      "FromBaseToUnitFunc": "x/3600d",
+      "Prefixes": [ "Kilo" ],
```

Comment:
Any variants of area to consider; squarecentimeter etc?

Reply:
Same as above.

Reply:
That is a perfectly valid reason. We only want to add units that someone actually wants or needs.